starboard/nplb/nplb_evergreen_compat_tests/sabi_test.cc

```diff
@@ -197,6 +197,55 @@ const char* kSabiJsonIdX64Sysv =
     "\"target_arch\":\"x64\",\"target_arch_sub\":\"\",\"word_size\":64}";
 #endif  // SB_API_VERSION == 16
 
+#if SB_API_VERSION == 17
+const char* kSabiJsonIdArmHardfp =
+    "{\"alignment_char\":1,\"alignment_double\":8,\"alignment_float\":4,"
+    "\"alignment_int\":4,\"alignment_llong\":8,\"alignment_long\":4,"
+    "\"alignment_pointer\":4,\"alignment_short\":2,\"calling_convention\":"
+    "\"eabi\",\"endianness\":\"little\",\"floating_point_abi\":\"hard\","
+    "\"floating_point_fpu\":\"vfpv3\",\"sb_api_version\":16,\"signedness_of_"
```

Comment:
You'll have to update the sb_api_version here as well

Reply:
I did, it grabs the value from https://github.com/youtube/cobalt/pull/3884/files#diff-989138430bc663c7b2bb63806b3c9252735f567838cf7a6438626ea1aa1ea4b3R9
Missed it in the first set of changes, so it's rebuilding rn.